dev/devicelab/README.md

```diff
@@ -4,6 +4,60 @@ This package contains the test framework and tests that run on physical devices.
 More generally the tests are referred to as "tasks" in the API, but since we
 primarily use it for testing, this document refers to them as "tests".
 
+You can see the continuous build results from the master branch at
+http://go/flutter-dashboard/build.html.
+
+# Running tests locally
+
+Do make sure your tests pass locally before deploying to the CI environment.
+Below is a handful of commands that run tests in a fashion very close to how the
+CI environment runs them. These commands are also useful when you need to
+reproduce a CI test failure locally.
+
+To run a test use option `-t` (`--task`):
+
+```sh
+dart bin/run.dart -t {NAME_OF_TEST}
+```
+
+To run multiple tests repeat option `-t` (`--task`) multiple times:
+
+```sh
+dart bin/run.dart -t test1 -t test2 -t test3
+```
+
+To run all tests defined in `manifest.yaml` use option `-a` (`--all`):
+
+```sh
+dart bin/run.dart -a
+```
+
+To run tests from a specific stage use option `-s` (`--stage`):
+
+```sh
+dart bin/run.dart -s {NAME_OF_STAGE}
+```
+
+# Reproducing broken builds locally
+
+So you noticed that a commit broke one of the tests. On
```

Comment:
Revise: "If a commit caused a test to fail, the dashboard might look something like this:"